lib/awful/hotkeys_popup/widget.lua

```diff
@@ -55,7 +55,7 @@ local widget = {
 widget.hide_without_description = true
 
 --- Merge hotkey records into one if they have the same modifiers and
--- description.
+-- description. Records with five or more keys will abbreviate them.
```

Comment:
please mention what it's only for awful.keys but not manually added hotkeys for the apps

Reply:
Oh, right, sorry.

Just pushed another commit. Is 2d3f7e1e94b60d34cadf3e5ec531eca6884f38c6 okay?

Reply:
yup, thanks!